gadgets/trace_exec/test/integration/trace_exec_test.go

```diff
@@ -114,6 +112,7 @@ int main(int argc, char *argv[], char **envp) {
 	containerOpts := []containers.ContainerOption{
 		containers.WithContainerImage(containerImage),
 		containers.WithStartAndStop(),
+		containers.WithPrivileged(), // nsenter needs privileges
```

Comment:
Unfortunately we don't support creating privileged containers on k8s yet: https://github.com/inspektor-gadget/inspektor-gadget/blob/205b96d47d1eb9856d9079cbf929330130bcf8d9/pkg/container-utils/testutils/k8s.go#L64-L66

Is there any way to avoid it?

Reply:
Let's try with the setpriv command.